src/__tests__/project_workspace.test.js

```diff
@@ -11,9 +11,9 @@ import ProjectWorkspace from '../project_workspace';
 
 describe('setup', () => {
```

Comment:
this is the place I think we should test `pathToJest` is a shadow accessor instead of a real concrete property, while none of the other classes should only deal with `jestCommandLine`.